examples/notion-task-github-pr-sync/index.js

```diff
@@ -0,0 +1,140 @@
+/* ================================================================================
+
+	notion-task-github-pr-sync
+  
+  Glitch example: https://glitch.com/edit/#!/notion-github-sync
+  Find the official Notion API client @ https://github.com/makenotion/notion-sdk-js/
+
+================================================================================ */
+
+function print(x) {console.log(x)}
+const { Client } = require("@notionhq/client")
+const dotenv = require("dotenv")
+const { Octokit } = require("octokit")
+const _ = require("lodash")
+
+dotenv.config()
+const octokit = new Octokit({ auth: process.env.GITHUB_KEY })
+const notion = new Client({ auth: process.env.NOTION_KEY })
+
+const databaseId = process.env.NOTION_DATABASE_ID
```

Comment:
We don't use this `databaseId` - maybe we should check if the page is parented by this database before commenting on the page?